.github/action_scripts/send_transactions.js

```diff
@@ -0,0 +1,375 @@
+        lastRef
+    );
+
+    const accountSecondNode = dag4.createAccount();
+    accountSecondNode.loginSeedPhrase( FIRST_WALLET_SEED_PHRASE );
+    await accountSecondNode.connect(
+        {
+            networkVersion: '2.0',
+            l0Url: networkOptions.l0GlobalUrl,
+            l1Url: networkOptions.dagL1UrlSecondNode,
+            testnet: true
+        }
+    );
+    const signedTransaction2 = await accountSecondNode.generateSignedTransaction(
+        THIRD_WALLET_ADDRESS,
+        5000,
+        1,
+        lastRef
+    );
+
+    try {
+        await Promise.all( [
+            dag4.network.postTransaction( signedTransaction ),
+            dag4.network.postTransaction( signedTransaction2 )
+        ] );
+
+        logMessage( `Waiting ${SLEEP_TIME_UNTIL_QUERY}ms until fetch wallet balances` );
+        await sleep( SLEEP_TIME_UNTIL_QUERY );
+
+        const firstWalletBalance = await accountFirstNode.getBalanceFor( FIRST_WALLET_ADDRESS );
+        const secondWalletBalance = await accountFirstNode.getBalanceFor( SECOND_WALLET_ADDRESS );
+        const thirdWalletBalance = await accountFirstNode.getBalanceFor( THIRD_WALLET_ADDRESS );
+
+        if( secondWalletBalance === 9900 && thirdWalletBalance === 5000 ){
+            logMessage( `Amount sent to third wallet ${thirdWalletBalance} and not to second: ${secondWalletBalance}` );
+            logMessage( `FirstWalletBalance: ${firstWalletBalance}` );
+            logMessage( `SecondWalletBalance: ${secondWalletBalance}` );
+            logMessage( `ThirdWalletBalance: ${thirdWalletBalance}` );
+            return;
+        }
+
+        if( secondWalletBalance === 14900 && thirdWalletBalance === 0 ){
+            logMessage( `Amount sent to second wallet ${thirdWalletBalance} and not to third: ${secondWalletBalance}` );
+            logMessage( `FirstWalletBalance: ${firstWalletBalance}` );
+            logMessage( `SecondWalletBalance: ${secondWalletBalance}` );
+            logMessage( `ThirdWalletBalance: ${thirdWalletBalance}` );
+            return;
+        }
+
+        throw Error( `Double spend occurred` );
+    } catch( error ) {
+        const errorMessage = `Error when sending double spend transaction between wallets, message: ${error}`;
+        logMessage( errorMessage );
+        throw error;
+    }
+};
+
+const assertBalance = async (
+    originBalance,
+    destinationBalance,
+    isInitial
+) => {
+    const expectedOriginBalance = isInitial ? 8900 : 9900;
+    const expectedDestinationBalance = isInitial ? 11000 : 9900;
+
+    if(
+        Number( originBalance ) !== expectedOriginBalance ||
+        Number( destinationBalance ) !== expectedDestinationBalance
+    ) {
+        throw Error( `
+        Error sending transactions. Wallet balances are different than expected:
+        expectedOriginBalance: ${expectedOriginBalance} ---- originBalance: ${originBalance}
+        expectedDestinationBalance: ${expectedDestinationBalance} ---- ${destinationBalance}
```

Comment:
I'd write `destinationBalance:` in a log message as you did with `originBalance:`